mmedit/core/evaluation/metrics.py

```diff
@@ -215,6 +215,52 @@ def psnr(img1, img2, crop_border=0, input_order='HWC', convert_to=None):
     return 20. * np.log10(255. / np.sqrt(mse_value))
 
 
+def mae(img1, img2, crop_border=0, input_order='HWC', convert_to=None):
+    """Calculate mean average error for evaluation.
+
+    Args:
+        img1 (ndarray): Images with range [0, 255].
+        img2 (ndarray): Images with range [0, 255].
+        crop_border (int): Cropped pixels in each edges of an image. These
+            pixels are not involved in the PSNR calculation. Default: 0.
+        input_order (str): Whether the input order is 'HWC' or 'CHW'.
+            Default: 'HWC'.
+        convert_to (str): Whether to convert the images to other color models.
+            If None, the images are not altered. When computing for 'Y',
+            the images are assumed to be in BGR order. Options are 'Y' and
+            None. Default: None.
+
+    Returns:
+        float: mae result.
+    """
+
+    assert img1.shape == img2.shape, (
+        f'Image shapes are different: {img1.shape}, {img2.shape}.')
+    if input_order not in ['HWC', 'CHW']:
+        raise ValueError(
+            f'Wrong input_order {input_order}. Supported input_orders are '
+            '"HWC" and "CHW"')
+    img1 = reorder_image(img1, input_order=input_order)
+    img2 = reorder_image(img2, input_order=input_order)
+
+    img1, img2 = img1.astype(np.float32), img2.astype(np.float32)
+    img1, img2 = img1 / 255., img2 / 255.
+    if isinstance(convert_to, str) and convert_to.lower() == 'y':
+        img1 = mmcv.bgr2ycbcr(img1, y_only=True)
+        img2 = mmcv.bgr2ycbcr(img2, y_only=True)
```

Comment:
Should this function assume bgr input? Should it accept rgb too?

I suggest `convert_to` be some string like `'bgr2y'` or `'rgb2y'`, and the `mmcv.bgr2ycbcr` is retrieved from a `str` to `function` dict. 

```
func = convert_dict['rgb2y']
img1 = func(img1, y_only=True)
```